x-pack/plugins/encrypted_saved_objects/server/plugin.ts

```diff
@@ -48,7 +44,7 @@ export interface LegacyAPI {
  */
 export class Plugin {
   private readonly logger: Logger;
-  private savedObjectsSetup?: ReturnType<typeof setupSavedObjects>;
+  private savedObjectsSetup!: ReturnType<typeof setupSavedObjects>;
```

Comment:
**question** Is there a benefit or preference to using `ReturnType<typeof setupSavedObjects>` instead of the exported `SavedObjectsSetup` type?

Reply:
Hmm, looks like `ReturnType<typeof setupSavedObjects>` is just a stupid alternative for `SavedObjectsSetup`, have no idea why I have it here :see_no_evil: Good catch, will fix!